lib/commands/cryptographic-identities.ts

```diff
@@ -614,10 +624,19 @@ export class ImportCryptographicIdentity implements ICommand {
 				this.$errors.failWithoutHelp("You have reached the maximum number of authentication attempts for this operation.");
 			}
 
-			_.each(result, identity => {
-				this.$logger.info("Imported certificate '%s'.", identity.Alias);
-			});
-		}).future<void>()();
+			return result;
+		}).future<Server.CryptographicIdentityData[]>()();
+	}
+
+	private ImportCertificateWithoutPassword(importType: string, certificateFile: string): IFuture<Server.CryptographicIdentityData[]> {
+		return (() => {
+			try {
+				let targetFile = this.$fs.createReadStream(certificateFile);
+				return this.$server.identityStore.importIdentity(<any>importType, '', targetFile).wait();
+			} catch (error) {
+				this.$errors.failWithoutHelp(error.message);
```

Comment:
Else you get a message which more often than not says there is no request for this certificate after which you get the help for the command.
Given the fact that you've used the command properly I don't think it's suitable to display help